scanner/src/main/kotlin/provenance/NestedProvenanceScanResult.kt

```diff
@@ -198,6 +168,53 @@ data class NestedProvenanceScanResult(
     }
 }
 
+/**
+ * Merge the nested [ScanResult]s into one [ScanResult] per used scanner, using the provenance from the [ScanResult]
+ * associated with the empty path as provenance. This maps the given [scanResultsByPath] to the format currently used
+ * by [OrtResult]. When merging multiple [ScanSummary]s the earliest start time will be used as the new start time, and
+ * the latest end time will be used as the end time. Because the [ScanSummary] does not contain the checksums of the
```

Comment:
Based on your previous changes the comment should be adapter to clarify that "the earliest start time **for each scanner** will be used as the new start time" and similar for the end time. Same for the merge function in `NestedProvenanceScanResult`.